lib/array_intersection.rb

```diff
@@ -1,6 +1,63 @@
 # Returns a new array to that contains elements in the intersection of the two input arrays
-# Time complexity: ?
-# Space complexity: ?
+# Time complexity: O(n) where n is the length of the longer array
```

Comment:
This is actually incorrect.  The time complexity would be O(n + m) where n and m are the lengths of the two arrays.